### DISCUSSION TITLE
Keyboard shortcuts for Feature questions

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#179] [General Survey Design]
Low priority, but it may be nice to have 1, 2, 3 keyboard shortcuts for the feature questions.

**Comment by mikequiapos** [comment]
When creating shortcuts, it's important to remember a few things. First, don't use shortcuts that already do something on the computer or internet. Second, make sure people can easily find the shortcuts by having a special menu or a question mark icon. Third, make sure the shortcuts don't work when someone is typing. Fourth, don't change shortcuts that people already know and use. It's important to respect the shortcuts they are used to.
